ccollect

```diff
@@ -366,9 +451,20 @@ while [ "${source_no}" -lt "${no_sources}" ]; do
     # with error message.
     #
     lock "${name}"
+    if [ "${SIMULATE}" ]
+    then
```

Comment:
This is only in SIMULATE mode. No backup is performed. I used this for testing parallel behaviuor with upper limit of children. sleep simulates actual child's (busy) work. I use awk to generate random number. srand() is by defauly initialized with current date and time of the day. So I put one sleep before starting a child. Without it, initial num of children is started fast, within a second and each child is srand-ed with the same value so each child is busy for ~same amount of time. With this SIMULATE I could test max num of child processes control without actual backup. I left these lines of code, but we can also remove them. In normal operation of ccollect this SIMULATE will and should never be turned on.